### PR TITLE
Version 0.6.0

### PR DESCRIPTION
**Pull request by nia-e**
🥳 
I don't think I have rights to push a new release (or if so I'm not sure how) but this bumps version to 0.6

### CODE REVIEW

**Comment by nia-e** [comment]
Nevermind, figured it out

**Comment by nia-e** [comment]
@rafaelcaricio how did you get around the package size limit on crates.io? `lvgl-sys` comes out to 23MiB which is over the limit :c

File: lvgl-codegen/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lvgl-codegen"
-version = "0.5.2"
+version = "0.6.0"
 description = "Code generation based on LVGL source code"
 authors = ["Rafael Caricio <crates.lvgl@caric.io>"]
```
Comment (rafaelcaricio) [inline lvgl-codegen/Cargo.toml:5]: You can add your name in those.

**Comment by rafaelcaricio** [comment]
I've never hit any limits before 👀

**Comment by nia-e** [comment]
I figured it out in the end, just required a minor version bump (and another then to make it work on `docs.rs` :c)

File: lvgl-codegen/Cargo.toml (hunk 1 above)
Comment (nia-e) [reply lvgl-codegen/Cargo.toml:5]: :o ty!! I will for the next release then, thanks so much

**Pull request merged**